### ISSUE TITLE
Clojars Browser error while attempting to load project list

### ISSUE THREAD

**Issue by rheimbuch** [cloxp/cloxp-install#3]
Reporting here because the Clojars Browser appears to be stored(?) in `cloxp-install/PartsBin/Clojure/`.

Steps:
1. fresh checkout of cloxp-install at commit 85b85b8f
2. install cloxp: `lein run -m rksm.cloxp-installer.main "pre-0.0.8"`
3. start cloxp: `./start.sh`
4. open: `http://localhost:9001/cloxp.html`
5. open Clojars Browser: `Alt-x, 'Clojure: browse Clojars'`
6. error pop-up appears: `TypeError: projects.map is not a function`

I tracked this down to an issue in the `loadProjectListViaHTTP` script attached to the ClojarsBrowser:

``` javascript
  function getProjectFileName(n) {
    Global.clojure.Runtime.doEval(
      "(.getCanonicalPath (rksm.system-navigator.clojars/ensure-clojure-feed-in-a-file))",
      {requiredNamespaces: ["rksm.system-navigator.clojars"], resultIsJSON: false, passError: true}, n);
  }
```

The `resultsJSON: false` option causes `getProjectFileName` to return a doubly-quoted string: 

```
"\"/Users/ryan.heimbuch/src/cloxp-install/LivelyKernel/2015-05-02_13-clojars-feed.json\""
```

which later causes a 400 error when attempting to load the file: 

```
"GET /nodejs/CommandLineServer/read-file?mimeType=application/json&fileName=\%22/Users/ryan.heimbuch/src/cloxp-install/LivelyKernel/2015-05-02_13-clojars-feed.json\%22 HTTP/1.1" 400
```

Setting `resultsJSON: true` in `getProjectFileName` fixes this for me:

``` javascript
  function getProjectFileName(n) {
    Global.clojure.Runtime.doEval(
      "(.getCanonicalPath (rksm.system-navigator.clojars/ensure-clojure-feed-in-a-file))",
      {requiredNamespaces: ["rksm.system-navigator.clojars"], resultIsJSON: true, passError: true}, n);
  }
```

I noticed that evaluating `(.getCanonicalPath (rksm.system-navigator.clojars/ensure-clojure-feed-in-a-file))` in a clojure workspace also returns the same doubly-quoted string. I traced the evaluation code as far as the `Global.clojure.Runtime.processNreplEvalAnswers` function, but I'm not sure what the intended behavior is.

**Comment by rksm** [comment]
Thanks for the detailed report! Do you have any leiningen profile additions installed such as cider nrepl?

**Comment by rksm** [comment]
So yes, cider-nrepl changes how out and value properties are handled. Fix d9c61d3 will deactivate all plugins for now, I'll look into how just specific nrepl-middleware settings can be overridden

**Comment by rheimbuch** [comment]
Good to know. I do have cider-nrepl in my user profile. It hadn't occurred
to me that would affect the cloxp evaluation.
